advocacy_docs/partner_docs/DBeaverGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Main page should have DBeaver PRO and not just DBeaver.  in all the boxes.
Also guide name is Implimentation guide for DBeaver PRO

Reply:
Already have that taken care of in the index